### PR TITLE
create proposals.md

### PR DESCRIPTION
**Pull request by gingerkidney**
community members can find communication seminar schedule here.

### CODE REVIEW

**Comment by gingerkidney** [comment]
@shenli @lilin90 PTAL

**Comment by gingerkidney** [comment]
@XuHuaiyu PTAL

File: proposals.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Proposals
+This directory contains the schedule of communication seminar and design proposals.
+
+## Communicaiton Seminar
```
Comment (XuHuaiyu) [inline proposals.md]: s/ Communicaiton/Communication

File: proposals.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+A communication seminar will be organized to discuss the details after a proposal is reviewed. Community members are welcomed to attend.
+
+### Seminar Schedule
+The schedule contains all ongoing proposals, please find the information below and join the seminar if you are interested. 
+
+| Time | Proposal | Author | Issue | Seminar Link |
+|------|--------|--------|---------|----------|
+| Unsettled |[Implement View Feature](https://docs.google.com/document/d/1fzWxn9eqi4qwhS5KmIgQd-AhNt1J1HZfc4jf7HTSAzo/edit?ts=5bed45cc#)| Andrew Di |[Support for view feature #7974](https://github.com/pingcap/tidb/issues/7974) | Unsettled |
```
Comment (zz-jason) [inline proposals.md]: how about `s/Andrew Di/[AndrewDi](https://github.com/AndrewDi)/`

File: proposals.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# Proposals
+This directory contains the schedule of communication seminar and design proposals.
+
+## Communicaiton Seminar
+A communication seminar will be organized to discuss the details after a proposal is reviewed. Community members are welcomed to attend.
+
+### Seminar Schedule
+The schedule contains all ongoing proposals, please find the information below and join the seminar if you are interested.
```
Comment (CaitinChen) [inline proposals.md]:
```suggestion
The schedule contains all ongoing proposals, and please find the information below and join the seminar if you are interested. 
```

Comment (CaitinChen) [reply proposals.md]: Note: Please do not combine two sentences with a comma. ^_^ Instead, you can use a conjunction (e.g. "and") or just separate them with a period.

File: proposals.md (hunk 2 above)
Comment (shenli) [inline proposals.md]: The time is already scheduled now. Please remind the timezone.

**Comment by zz-jason** [comment]
LGTM

**Pull request merged**